libraries/Microsoft.Bot.Builder.AI.QnA/Utils/GenerateAnswerUtils.cs

```diff
@@ -153,7 +153,7 @@ private static void ValidateOptions(QnAMakerOptions options)
         private QnAMakerOptions HydrateOptions(QnAMakerOptions queryOptions)
         {
             var hydratedOptions = JsonConvert.DeserializeObject<QnAMakerOptions>(JsonConvert.SerializeObject(this.Options));
-
+            
             if (queryOptions != null)
             {
                 if (queryOptions.ScoreThreshold != hydratedOptions.ScoreThreshold && queryOptions.ScoreThreshold != 0)
```

Comment:
These if statements are too complex, and they require detailed knowledge of operator precedence. Please simplify.

Reply:
Simplified by removing unnecessary, if check condition.

---
In reply to: [430554363](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430554363) [](ancestors = 430554363)